src/components/BodyWithResults/index.ts

```diff
@@ -1,3 +1,9 @@
+import { connect } from 'react-redux';
+import { State } from '../../state/reducers';
 import BodyWithResults from './BodyWithResults';
 
-export default BodyWithResults;
+const mapStateToProps = (state: State) => ({
```

Comment:
How about destructing state?
```
const mapStateToProps = ({ isEmpty }: State) => ({ isEmpty });
```

Reply:
I don't like the idea to destruct props and state into variables, because it will be hard for me to understand what is this variable from during code review without the full file visibility